components/database/dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mysql:5.7
```

Comment:
I had to write a Dockerfile for database to prevent injecting schema which needs extra volume to store the schema and db init shell script.

Reply:
@kmehant can't we directly use mysql docker-image from dockerhub ?

Reply:
@kmehant can you please describe why we need to write Dockerfile as we can directly pull mysql docker-image from dockerhub.

Reply:
@demon-36 we can do that also, as I have said earlier we need extra volumes for mounting schema and init db script. It better to bind them in to a separate image and use in k8s